assembl/static2/js/app/components/administration/voteSession/customizeGaugeForm.jsx

```diff
@@ -207,9 +217,9 @@ export class DumbCustomizeGaugeForm extends React.Component<DefaultProps, Props,
             handleUnitChange={this.handleUnitChange}
             handleGaugeChoiceLabelChange={this.handleGaugeChoiceLabelChange}
           />
-          {/* <Checkbox onClick={this.toggleApplyToAllProposals}>
```

Comment:
forgotten commented lines :)

Reply:
it is a line removal :-) (in fact, I uncommented the line)

Reply:
Oups, guess I shouldn't CR before coffee